f8a_worker/dispatcher/flows/githubManifestMetadataFlow.yml

```diff
@@ -33,4 +33,6 @@
                    key: 'recursive_limit'
                    value: 0
         - from: 'dependency_snapshot'
-          to: 'GitHubManifestMetadataResultCollector'
\ No newline at end of file
+	  to:    'github_manifest_details'	
+	- from:    'github_manifest_details'
```

Comment:
Please do not mix tabs and spaces (can be misleading). Use spaces as we already use spaces.